content/events/agbt2025-advances-in-genome-biology-and-technology.md

```diff
@@ -0,0 +1,44 @@
+---
+author: "AnVIL"
+conference: "AGBT 2025"
+description: "Advances in Genome Biology and Technology (AGBT) delivers a premier experience where heads of labs, institutions, businesses, financial analysts and other high-level stakeholders come together to advance the field and drive game-changing innovation."
+eventType: "Conference"
+featured: true
+location: "Marco Island, FL, USA"
+sessions:
+  [
+    { sessionEnd: "23 Feb 2025 5:00 PM", sessionStart: "23 Feb 2025 9:00 AM" },
+    { sessionEnd: "24 Feb 2025 5:00 PM", sessionStart: "24 Feb 2025 9:00 AM" },
+    { sessionEnd: "25 Feb 2025 5:00 PM", sessionStart: "25 Feb 2025 9:00 AM" },
+    { sessionEnd: "26 Feb 2025 5:00 PM", sessionStart: "26 Feb 2025 9:00 AM" },
+  ]
+
+timezone: "America/New_York"
+title: "Advances in Genome Biology and Technology 2023"
```

Comment:
@esthermcdade I believe the title should be `Advances in Genome Biology and Technology 2025`. Update here and also in the `.mdx` file.